src/wms/src/main/java/org/geoserver/wms/capabilities/Capabilities_1_3_0_Transformer.java

```diff
@@ -669,9 +669,13 @@ private void handleLayers() {
             final Catalog catalog = wmsConfig.getCatalog();
                         
             WMSInfo serviceInfo = wmsConfig.getServiceInfo();
-            element("Title", serviceInfo.getTitle());
-            element("Abstract", serviceInfo.getAbstract());
-
+            if(StringUtils.isBlank(serviceInfo.getRootLayerTitle())) {
```

Comment:
The two settings are independent, it might be that someone sets only the abstract but not the title, I'd separate the two checks.